src/styles/partials/_main.scss

```diff
@@ -0,0 +1,136 @@
+@use "./variables" as *;
+
+//  **********************************😀😀 MOVIES STYLES 😀😀***************************
+
+.movies-container {
+  color: $secondary;
+}
+.movies-wrapper {
+  max-width: 1500px;
+  min-width: 1500px;
+  margin: 0 auto;
+}
+.title {
+  font-size: $title-font-size;
+  margin: 10px 0 0 70px;
+  padding: 2px;
+}
+.movies {
+  display: flex;
+  width: 100%;
+  overflow-x: auto;
+  margin-left: 70px;
+  padding: 2px;
+  scrollbar-width: none; /* Hide scrollbar in Firefox */
+  -ms-overflow-style: none; /* Hide scrollbar in Internet Explorer and Edge */
+}
+.movies::-webkit-scrollbar {
+  display: none;
+}
+.movie {
+  min-width: 200px;
+  height: 300px;
+  transition: transform 450ms;
+  overflow-y: hidden;
+  will-change: transform;
+  margin-right: 10px;
+  cursor: pointer;
+  &:hover {
+    transform: scale(1.08);
+    opacity: 1;
+  }
+  img {
+    height: 100%;
+  }
+}
+
+.movies-results .wrapper {
+  display: flex;
+  .movies {
+    width: 100%;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    overflow-x: hidden;
+    margin-left: 0px;
+  }
+  .movie {
+    min-width: 200px;
+    height: 300px;
+
+    transition: transform 450ms;
+    overflow-y: hidden;
+    margin: 20px 10px;
+
+    cursor: pointer;
+    &:hover {
+      transform: scale(1.08);
+      opacity: 1;
+    }
+    img {
+      height: 100%;
+    }
+  }
+}
+
+.no-data-found .wrapper {
+  display: flex;
+  color: $secondary;
+  opacity: 0.7;
+  margin-top: 30px;
+  justify-content: center;
+}
+
+@media (max-width: 992px) {
+  .title {
+    margin: 20px 0 0 30px;
+  }
+  .movies {
+    margin-left: 30px;
+  }
+}
+
+@media (max-width: 768px) {

```

Comment:
the searched movies looked slightly squished , perhaps you could try making it height 100%

Reply:
just fixed this one thanks